src/actions/peerActions.ts

```diff
@@ -0,0 +1,39 @@
+export enum ConnectionStatus {
+  Connected = 'connected',
+  Disconnected = 'disconnected',
+}
```

Comment:
It would be nice to be unified with one rule.
(Naming convention of enum)

PeerActionTypes is CONSTANT_CASE (CONNECT_PEER, DISCONNECT_PEER)
whereas ConnectionStatus is PascalCase (Connected, Disconnected).